app/assets/stylesheets/application.scss

```diff
@@ -17,3 +17,6 @@
 
 // App-specific custom styles and overrides
 @import "local/custom";
+
+// Sorting styles for table headers
+@import "local/sorting";
```

Comment:
Tiny thing but components is best to all go above the `@import "local/custom";` declaration, leaving this one as the last one, because that way, without having to touch the components itself, we can tweak some things by using the local/custom file.